main.go

```diff
@@ -71,6 +73,8 @@ func init() {
 	utilruntime.Must(opv1a1.AddToScheme(scheme))
 	utilruntime.Must(extv1.AddToScheme(scheme))
 	utilruntime.Must(quotav1.AddToScheme(scheme))
+	utilruntime.Must(nbapis.AddToScheme(scheme))
+	utilruntime.Must(corev1.AddToScheme(scheme))
```

Comment:
doesn't clientgoscheme already registers kube native APIs?

Reply:
yes, it does removed corev1 from scheme in new commit